arts/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+
+from core.models import SoftDeleteModel
+
+class Art(SoftDeleteModel):
+    title        = models.CharField(max_length=30)
+    image_url    = models.URLField()
+    description  = models.TextField()
+    price        = models.DecimalField(max_digits=8, decimal_places=2)
+    is_available = models.BooleanField(default=True)
+    artist       = models.ForeignKey('Artist', on_delete=models.CASCADE)
+    size         = models.ForeignKey('Size', on_delete=models.CASCADE)
+    shape        = models.ForeignKey('Shape', on_delete=models.CASCADE)
+    themes       = models.ManyToManyField('Theme', through='Art_Theme', related_name='arts')
+    colors       = models.ManyToManyField('Color', through='Art_Color', related_name='arts')
```

Comment:
Class명은 `CamelCase`를 사용해서 만들어주세요

```suggestion
    themes       = models.ManyToManyField('Theme', through='ArtTheme', related_name='arts')
    colors       = models.ManyToManyField('Color', through='ArtColor', related_name='arts')
```

Reply:
수정하엿습니다